docs/integrations/keras_integration.rst

```diff
@@ -0,0 +1,597 @@
+
+Neptune-Keras Integration
+*************************
+
+What will you get with this integration
+=======================================
+
+In this tutorial, you'll learn how to integrate Neptune into your Keras application, and at the end you'll be able to efficiently track and monitor model training, metrics, images and even datasets.
```

Comment:
This section is about what the integration gives you (logging metrics, hardware etc) not about the tutorial itself. 

Please adjust accordingly.